modules/geo-layers/src/mesh-layer/mesh-layer.js

```diff
@@ -17,7 +18,8 @@ function validateGeometryAttributes(attributes) {
 
 const defaultProps = {
   // PBR material object. _lighting must be pbr for this to work
-  pbrMaterial: {type: 'object', value: null}
+  pbrMaterial: {type: 'object', value: null},
+  featureIds: {type: 'Uint32Array', value: null}
```

Comment:
This is not a known type: https://deck.gl/docs/developer-guide/custom-layers/prop-types#built-in-types

Just set it to `object` since you don't need any special comparison logic.

Reply:
Ok, actually it is an `array` type.